lib/sqlalchemy_ingres/base.py

```diff
@@ -641,7 +643,7 @@ def denormalize_name(self, name):
         if name is None:
             return None
         else:
-            return name.lower().encode('latin1')
```

Comment:
I'm guessing but I suspect this is accidentally working for the test case (compare with normalize name which does NOT change case). I recommend reverting this line and then making this separate and handling DB_NAME_CASE lookup.

Reply:
Additionally, the existing code with encoding change is DEFINTELY incorrect. This will only work for some values/II_CHARSETxx settings/combinations. This needs a separate issue, I just opened https://github.com/ActianCorp/sqlalchemy-ingres/issues/36 (worth a jira ticket too for cross reference?)

Reply:
Reverted change and will deal with it in #36